File: day1.js
fs = require("fs");
fs.readFile("input/day1.sql", "utf8", (err, data) => {
  let listData = data.split("\n\n");
  listData = listData.map((i) =>
    i.split("\n").reduce((a, b) => parseInt(a) + parseInt(b), 0)
  );
  listData.sort((a, b) => b - a);
  console.log("A) Max elf:", listData[0]);
  console.log(
    "B) Max elf:",
    listData.slice(0, 3).reduce((a, b) => a + b)
  );
});

File: day1.py
with open("input/day1.sql", 'r') as f:
    elfs = [sum([int(j) for j in i.split('\n') if j]) for i in f.read().split('\n\n')]
    elfs.sort(reverse=True)
    print(f"A) max elf:    {elfs[0]}")
    print(f"B) max 3 elfs: {sum(elfs[0:3])}")
